lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3516,6 +3516,20 @@ namespace {
             Impl.ImportedDecls.end())
           continue;
 
+        // If we encounter an IndirectFieldDecl, ensure that its parent is
+        // importable before attempting to import it because they are dependent
+        // when it comes to getter/setter generation.
+        if (const auto *ind = dyn_cast<clang::IndirectFieldDecl>(nd)) {
+          const clang::CXXRecordDecl *ParentUnion =
```

Comment:
`ParentUnion` -> `parentUnion`.

(Sorry to be pedantic.)

Reply:
Ah, no problem totally fine. I am still adjusting from the LLVM starts with caps style.

Reply:
done

Reply:
No worries, thanks.